blackbox/docs/protocols/postgres.txt

```diff
@@ -145,6 +145,38 @@ implemented::
     SHOW 1 row in set (... sec)
 
 
+Client compatibility
+====================
+
+JDBC
+----
+
+The `pgjdbc`_ JDBC driver starting with version ``9.4.1209`` can be used with
+Crate.
+
+
+Limitations
+~~~~~~~~~~~
+
+- *reflection* methods like ``conn.getMetaData().getTables(...)`` will not work
+  since they require postgres specific tables which do not exist in Crate.
+  As a workaround it is possible to query the standard ``information_schema``
+  tables manually using ``SELECT`` statements.
+
+- ``OBJECT`` and ``GEO_SHAPE`` can be streamed as ``JSON`` but require a
+  unreleased version of `pgjdbc`_.
+
+
+- ``executeBatch`` of ``PreparedStatement`` may continue even with individual
```

Comment:
i think we should point out that the difference in behaviour is the "not normal" case. for me this sounds like as if the continuation is not right, which is not the case. so in the end it just means that the user has to check the exception if he wants to know what was executed and what not